For 'Latest' release track - remove "formerly known as Versionless" (#8385)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/cloud-release-tracks.md

| Release track | Description | Plan availability | API value |
| ------------- | ----------- | ----------------- | --------- |

> | **Latest Fusion** <Lifecycle status="private_preview" /> | The latest build of the new engine for dbt, available to select accounts. | All plans | `latest-fusion` |

Author: Fusion was entirely missing from the page about Release Tracks. I added a quick mention of it in the table here, but the page probably warrants an overall revision. Rereading the FAQs further down, they reflect our state of mind circa 2024 more so than on-the-ground reality today

| Release track | Description | Plan availability | API value |
| ------------- | ----------- | ----------------- | --------- |

> | **Latest Fusion** <Lifecycle status="private_preview" /> | The latest build of the new engine for dbt, available to select accounts. | All plans | `latest-fusion` |

Reviewer: ```suggestion
| **Latest Fusion** <br /><Lifecycle status="private_preview" /> | The latest build of the new engine for dbt, available to select accounts. | All plans | `latest-fusion` |
```

> | **Latest Fusion** <br /><Lifecycle status="private_preview" /> | The latest build of the new engine for dbt, available to select accounts. | All plans | `latest-fusion` |

Reviewer: The spacing is weird here so I'm trying something
```suggestion
| **Latest Fusion**  | The latest build of the new engine for dbt, available to select accounts. | All plans <br /><Lifecycle status="private_preview" /> | `latest-fusion` |
```

> | **Latest** | Provides a continuous release of the latest functionality in the dbt platform.<br /><br />Includes early access to new features of the dbt framework before they're available in final releases of dbt Core. | All plans | `latest` |

Reviewer: Removing that "Final release" language so it doesn't sow confusion
```suggestion
| **Latest** | Provides a continuous release of the latest functionality in the <Constant name="dbt_platform" /> .<br /><br />Includes early access to new features of the dbt framework before they're available in <Constant name="core" />. | All plans | `latest` |
```
